readme.txt

```diff
@@ -6,7 +6,7 @@ Tested up to: 6.8.2
 Requires PHP: 7.4
 WC requires at least: 5.5.1
 WC tested up to: 10.1.2
-Stable tag: 4.2.1
+Stable tag: 4.2.2
```

Comment:
Issue: This should probably be updated from the main branch to avoid issues when we do a PR